AdminUI/admin-ui-app/README.md

```diff
@@ -0,0 +1,27 @@
+# AdminUiApp
+
+This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 15.2.0.
```

Comment:
Remove this line. It's not interesting for the reader

Reply:
This file is also generated by default by angular. I have removed the lines in question.